src/Heatmap/GoogleMapsUtils-Swift.h

```diff
@@ -0,0 +1,246 @@
+// Generated by Apple Swift version 5.2.4 (swiftlang-1103.0.32.9 clang-1103.0.32.53)
```

Comment:
I don't think this file is supposed to be checked in as it is a generated file at compile-time. Can you confirm?

Reply:
This file is created when you build the GoogleMapsUtils Xcode project. When users import GoogleMapsUtils, is the Xcode project responsible for creating the framework run?

Reply:
I think this file should be removed—you mentioned you added this manually from the generated .framework file, correct? If so, this should be removed. Same comment on the other `GoogleMapsUtils-Swift.h` file in the root directory.

Reply:
Ok I believe that's done! I'm just going to sanity check that the framework still works.

Reply:
Edit: it looks good!